app/rocket/kinetics.py

```diff
@@ -0,0 +1,90 @@
+from threading import Thread
```

Comment:
we should use multiprocessing/Process instead since it will give us a dedicated process instead of a thread

Reply:
here is a good explanation by [practico engineering](https://medium.com/practo-engineering/threading-vs-multiprocessing-in-python-7b57f224eadb)